Client/mods/deathmatch/logic/lua/CLuaManager.cpp

```diff
@@ -11,6 +11,7 @@
 
 #include "StdInc.h"
 #include "../luadefs/CLuaFireDefs.h"
+#include "../luadefs/CLuaPathFindDefs.h"
```

Comment:
As you removed your stuff about path find functions, remove that line as well.

Reply:
good i have you